data-reconciliation-app/demo/scripts/02-report_change_demo.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -euo pipefail
+
+VARS_PATH="./demo/scripts/env_vars.sh"
+. "$VARS_PATH"
+
+cd "${certs}"
+
+printf "Member1 trying to ingest additional data \n" 
+check_eq "Member1 - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg member0) -H "Content-Type: application/json" --data-binary "@../../demo/data/member1_demo_pt2.json" $only_status_code)"
+addCheckpoint "✅ Member1 successfully ingested additional data"
+
+curl $reportUrl -X GET $(cert_arg member0) |jq >> ../../demo/Reports/Member1_Report_Change_V1.json
+addCheckpoint "✅ Member 0 - read data report"
```

Comment:
here I see an inconsistency related to my previous comment regarding naming the members.
line 11: Member1
line 14: Member0

both related to the same member who is interacting with the app

Reply:
^this - I was trying to understand the flow as we have a zero indexed array, but then talk about Member1->3 :-) Bit confusing!

Reply:
I changed it to Member 1,2,3 as per the demo-guidance.md file.
But to avoid the confusion I will change it to Member 0,1,2.